src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -210,7 +210,7 @@ static void PrintSettings(IEnumerable<KeyValuePair<string, object>> settings)
                             }
                         }
 
-                        await Task.Delay(1).ConfigureAwait(false);
+                        await Task.Yield();
```

Comment:
@timbussmann `Task.Delay` uses basically a [timer and a task completion source](https://referencesource.microsoft.com/#mscorlib/system/threading/Tasks/Task.cs,5905). `Task.Yield` uses the more lightweight [`YieldAwaitable`](https://referencesource.microsoft.com/#mscorlib/system/runtime/compilerservices/YieldAwaitable.cs).